components/wishlist-page/wishlist-page.html

```diff
@@ -0,0 +1,165 @@
+
+                    <div class="cart-click-btns">
+                        <button class="btn-cart">Remove from wishlist</button><br>
+                        <button class="btn-cart">Add to cart</button>
+                    </div>
+
+                </section>
+            </div>
+        </div>
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
+                </section>
+                <section class="text-container">
+                    <div class="card-text card8-text">
+                        <header>
+                            <h3>T-shirt</h3><button class="rate-btn"><span>&#9733</span>4.5</button>
+                        </header>
+                        <div class="card-info">
+                            <p>High comfort cotton</p>
+                            <div class="price">Rs. 300
+                                <span class="price-old">RS. 600</span>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="cart-click-btns">
+                        <button class="btn-cart">Remove from wishlist</button><br>
+                        <button class="btn-cart">Add to cart</button>
+                    </div>
+
+
+                </section>
+            </div>
+        </div>
+
+        <div class="wishlist-card">
+            <div class="card card5 card7">
+                <section class="img-container">
+                    <img class="card-image" alt="card-image"
+                        src="https://images.unsplash.com/photo-1583743814966-8936f5b7be1a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8dCUyMHNoaXJ0fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=400&q=60">
+                </section>
+                <section class="text-container">
+                    <div class="card-text card8-text">
+                        <header>
+                            <h3>T-shirt</h3><button class="rate-btn"><span>&#9733</span>4.5</button>
+                        </header>
+                        <div class="card-info">
+                            <p>High comfort cotton</p>
+                            <div class="price">Rs. 300
+                                <span class="price-old">RS. 600</span>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="cart-click-btns">
+                        <button class="btn-cart">Remove from wishlist</button><br>
+                        <button class="btn-cart">Add to cart</button>
+                    </div>
+
+                </section>
+            </div>
+        </div>
+
+    </section>
+
+
+
```

Comment:
remove spacing.

Reply:
I have made the change,thnks for reviewing